cdisc_rules_engine/services/data_readers/data_reader_factory.py

```diff
@@ -20,6 +22,7 @@ def register_service(cls, name: str, service: Type[DataReaderInterface]):
         """
         Registers a new service in internal _service_map
         """
+        print("registering data reader service with name = ", name)
```

Comment:
Remove print statements. If something needs to be logged it should use the logger